javascript-loops-practice/odd-or-even.js

```diff
@@ -0,0 +1,14 @@
+/* exported oddOrEven */
+function oddOrEven(array) {
+  var newArray = [];
+  var i = array.length;
+  while (i > 0) {
+    if (array[i - 1] % 2 === 0) {
+      newArray.unshift('even');
+    } else {
+      newArray.unshift('odd');
+    }
+    i--;
+  }
```

Comment:
Same feedback here, there's no real reason this loop needs to be going backwards, please make it move forwards through the array.